account_menu/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+* Suggest to rename to something like `account_usability` in 15.0, given that
```

Comment:
I also agree with this recommendation

Reply:
Understood. I don't think a renaming is needed since menus are still the main functionality.

Reply:
Okay, they are the menu feature, however the module has more features than the menu, so the name change could give more sense to the rest of the added features, what do you think?

Reply:
I see your point. I'll make sure to make those changes asap.

Reply:
@ChrisOForgeFlow It should be good now.